core/src/raw/path_cache.rs

```diff
@@ -198,9 +206,22 @@ impl<Q: PathQuery> PathCacher<Q> {
     }
 }
 
+/// Normalize the path for dirs
+pub fn normalize_dir_path(path: &str) -> &str {
```

Comment:
Hi, I feel like this change should only be inside gdrive. We don't need to touch `raw`.

Reply:
makes sense! updated